apps/publisher/themes/default/theme.js

```diff
@@ -304,9 +304,9 @@ var engine = caramel.engine('handlebars', (function() {
             };
 
 
-            var renderOptionsForOptionsText = function(value, values, field) {
+            var renderOptionsForOptionsText = function(value, values, field,mode) {
                 var id=field.name.tableQualifiedName+'_option';
-                var out = '<select ' + renderFieldMetaData(field,id) + '>';
+                var out = '<select ' + renderFieldMetaData(field,id,mode) + '>';

```

Comment:
Thanks, I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/1b1eb4bc8b20375140097c9ce10e1c9bea4b97c0)